shigatyper/2.0.1/Dockerfile

```diff
@@ -81,5 +81,32 @@ WORKDIR /data
 
 FROM app as test
 
-COPY test.sh .
-RUN bash test.sh
+# install fastq-dl via conda for downloading data
+# instead installing fastq-dl from bioconda and using that to get the fastq files for testing
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
```

Comment:
I think we should avoid installing things in the test phase, because it's possible that something installed in the test phase is what is getting the tool to work.

Reply:
agreed. honestly did it because I strongly dislike sra-toolkit

Reply:
and I'd prefer not to keep fastq files in this repo if possible

Reply:
You dislike the sra-toolkit, so you installed conda to install fastq-dl which uses fasterq-dump from the sra-toolkit?

Reply:
I re-looked up how to just use wget with ENA

```
# something like this _should_ work
wget ftp.sra.ebi.ac.uk/vol1/fastq/SRR818/005/SRR8186675/SRR8186675_1.fastq.gz && \
wget ftp.sra.ebi.ac.uk/vol1/fastq/SRR818/005/SRR8186675/SRR8186675_2.fastq.gz && \
wget ftp.sra.ebi.ac.uk/vol1/fastq/SRR773/008/SRR7738178/SRR7738178_1.fastq.gz && \
wget ftp.sra.ebi.ac.uk/vol1/fastq/SRR773/008/SRR7738178/SRR7738178_2.fastq.gz
```